src/vocation.h

```diff
@@ -116,6 +120,8 @@ class Vocation
 
 		uint8_t soulMax = 100;
 		uint8_t clientId = 0;
+
+		bool pvp = true;
```

Comment:
dunno, `canPvp`, `hasPvp`, `enabledPvp`, `allowPvp`, `somethingsomething`, `canEngagePvp`...

Reply:
same goes for flag name, im keeping this as draft and awaiting ideas for a nice property, flag and method names 😎

Reply:
From most to least I like `canEngagePvp`, `allowPvp`, and `enabledPvp`.

Reply:
And how do you like the one I went with?